CMakeLists.txt

```diff
@@ -448,28 +398,18 @@ add_subdirectory(third_party/abseil-cpp EXCLUDE_FROM_ALL)
 add_subdirectory(third_party/flatcc EXCLUDE_FROM_ALL)
 add_subdirectory(third_party/vulkan_headers EXCLUDE_FROM_ALL)
 
-if(CMAKE_CROSSCOMPILING)
-  # We need flatc to generate some source code. When cross-compiling, we need
-  # to make sure the flatc binary is configured under host environment.
-  iree_declare_host_excutable(flatc "flatc" BUILDONLY)
-  iree_declare_host_excutable(flatcc_cli "flatcc_cli" BUILDONLY)
-
-  # Set the FLATBUFFERS_FLATC_EXECUTABLE. It controls where to find the flatc
-  # binary in BuildFlatBuffers().
-  iree_get_executable_path(FLATBUFFERS_FLATC_EXECUTABLE flatc)
-
-  # Add a custom target to copy the flatc to the binary directory.
-  add_custom_target(iree_host_flatcc_cli
-    COMMAND
-      "${CMAKE_COMMAND}" -E copy_if_different
-        "${CMAKE_CURRENT_SOURCE_DIR}/third_party/flatcc/bin/flatcc${IREE_HOST_EXECUTABLE_SUFFIX}"
-        "${IREE_HOST_BINARY_ROOT}/bin/flatcc_cli${IREE_HOST_EXECUTABLE_SUFFIX}"
-    DEPENDS iree_host_build_flatcc_cli
-    COMMENT "Installing host flatcc..."
-  )
-else()
-  # TODO: unify flatc and flatcc handling to the same mechanism.
-  add_executable(iree_host_flatcc_cli ALIAS flatcc_cli)
+# Configure flatcc_cli's executable for host/target builds:
+#   "flatcc_cli" target name (when building for host)
+#   "flatcc_d" executable name (when referencing from target build)
```

Comment:
confused about this - why two names? why d?

Reply:
Good question >_>, which I would also like the answer to...

https://github.com/dvidelabs/flatcc/blob/master/src/cli/CMakeLists.txt

```
D:\dev\projects\flatcc (master -> origin)
λ cmake --build build/ --target flatcc_cli
[28/28] Linking C executable ..\bin\flatcc_d.exe
```

Reply:
Ah, also testing on Linux now:

```
+ /usr/bin/cmake --install .
-- Install configuration: "Release"
-- Installing: /usr/local/bin/flatcc
CMake Error at cmake_install.cmake:52 (file):
  file INSTALL cannot copy file
  "/usr/local/google/home/scotttodd/code/iree/third_party/flatcc/bin/flatcc"
  to "/usr/local/bin/flatcc": Permission denied.
```

Reply:
Annoying:
https://github.com/dvidelabs/flatcc/blob/0d47180eabc55ecd7a9f865477f27add74111da7/CMakeLists.txt#L303-L307

weird that it's not flatcc_cli_d tho

Reply:
Bleh. Discussed a bit on Discord. I added a new `iree-flatcc-cli` binary target. Could still be improved.